pulpcore/cli/ansible/context.py

```diff
@@ -0,0 +1,59 @@
+from pulpcore.cli.common.context import (
+    PulpEntityContext,
+    PulpRepositoryContext,
+    PulpRepositoryVersionContext,
+)
+
+
+# TODO Add Role and Collection Content contexts
+
+
+class PulpAnsibleDistributionContext(PulpEntityContext):
+    ENTITY = "distribution"
+    HREF = "ansible_ansible_distribution_href"
+    LIST_ID = "distributions_ansible_ansible_list"
+    READ_ID = "distributions_ansible_ansible_read"
+    CREATE_ID = "distributions_ansible_ansible_create"
+    UPDATE_ID = "distributions_ansible_ansible_partial_update"
```

Comment:
No it works both ways. I changed it due to an earlier comment saying PATCH was better. PATCH eases the requirements for the command allowing for the update syntax to be cleaner. With PUT, required fields would need to be re-entered even if you weren't updating them, or you would have to specified them to be the same value in the code in order to work like a PATCH command. I'll make a PR to change it everywhere since I think it's better.

Reply:
> PATCH eases the requirements for the command allowing for the update syntax to be cleaner.

Not sure i understand that sentence. But i agree that PATCH is probably safer and easier. I just want to have it used consistently in the whole project. And that change should not be done in this PR under the hood.